src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -181,7 +181,8 @@ def load_arguments(self, _):
         c.argument('disk_mbps_read_only', type=int, help='The total throughput (MBps) that will be allowed across all VMs mounting the shared disk as ReadOnly. MBps means millions of bytes per second - MB here uses the ISO notation, of powers of 10')
         c.argument('image_reference', help='ID or URN (publisher:offer:sku:version) of the image from which to create a disk')
         c.argument('image_reference_lun', type=int, help='If the disk is created from an image\'s data disk, this is an index that indicates which of the data disks in the image to use. For OS disks, this field is null')
-        c.argument('gallery_image_reference', help='ID of the Compute Gallery image version from which to create a disk')
+        c.argument('gallery_image_reference', help='ID of the Compute, Shared or Community Gallery image version from which to create a disk')
+        c.ignore('gallery_image_reference_backend')
```

Comment:
Would the naming be more accurate to change the `backend` to `type`?
```suggestion
        c.ignore('gallery_image_reference_type')
```